src/main/scala/temple/generate/server/go/service/dao/GoServiceDAOInputStructsGenerator.scala

```diff
@@ -17,6 +17,7 @@ object GoServiceDAOInputStructsGenerator {
     operation match {
       case List                            => s"read a ${root.decapitalizedName} list"
       case Create | Read | Update | Delete => s"${operation.toString.toLowerCase} a single ${root.decapitalizedName}"
+      case Identify                        => s"identify the single ${root.decapitalizedName}"
```

Comment:
"identify the current user" rather than "identify the single user"?

Reply:
Yes that's much nicer